workshops/fall-2019/outline.md

```diff
@@ -0,0 +1,33 @@
+Julia-Users-Group Fall Workshop
+TODO: insert date
+TODO: insert location
+
+This workshop is for everybody who is interested in learning Julia all levels welcome.
+Pre-requirements: You read our Code of Conduct (http://www.opentechschool.org/code-of-conduct/) and appreciate the content.
+
+10 am: We will give a quick introduction to the language
+1) Key features of the language and differences to other languages
+2) Which applications are particularly promising
+3) How to get started programming in Julia
+
+10.45 am: We split into balanced groups, depending on your previous experience (with similar languages)
+and your interests and we will offer a wide range of learning material.
+More experienced Julia users will help you whenever you get stuck.
+
+2 pm: We will be a bit more creative and work on several projects, e.g.
+- writing our own opentechschool tutorial(s)
+- implementing your current project into Julia
+- working on more advanced stuff, e.g.  https://etherpad.net/p/Julia_teaching-material
+
+Why Julia? -- Julia is Open Source. Julia is integrative: bloody beginners can use it, but also experts often prefer it over C or Fortran.
+It is nearly as fast as C, but as easy to program as Python. In recent years, many computer and data scientists have switched to Julia 
+as it seems to be the most future-proof language for large numerical operations.
+Today, Julia nicely complements Python and R when not only the comfort of an open source language that is easy to read and write
+but also speed is required.
+
+What is Julia made for? -- Julia is a general purpose language like Python, but it shines particularly when heavy computation is necessary
+and the alternatives are either to slow or require a lot of time to be implemented, e.g. data science, engineering, differential equations. Moreover, package development for Julia is rather convenient so that many scientists have switched to implement their packages in Julia.
+
+Will Julia be the future? -- According to the [Julia developer survey](https://julialang.org/images/2019-julia-user-developer-survey.pdf) Julia users are convinced it is.
+Julia reached version 1.0 in summer 2018 which indicates a stable programming experience and it is rapidly improving.
+
```

Comment:
I would be more inclined to do a bulletpoint summary of why to use Julia today. I know you've addressed many of these points in the text above, but it's just a tidier takeaway than the results of a survey.

Reply:
I don't have any strong opinion on this. Do you have an example of a bulletpoint list, that would fit?
Nevertheless I think it is good to link to the survey somewhere.